src/pfe/file-watcher/server/test/scripts/setup.sh

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env bash
+
+# Colors for success and error messages
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+BLUE='\033[0;36m'
+RESET='\033[0m'
+
+CW_DIR=~/codewind
```

Comment:
Can we also configure where to clone Codewind repo? If for example, a user has Codewind repo already in `~/codewind` then it cannot do `git clone $CODEWIND_REPO -b $TEST_BRANCH \
    && $CW_DIR/run.sh`

Reply:
or check if there is codewind already present, skip and use the existing codewind run.sh

Reply:
This case won't happen because we call uninstall function before install function when user uses our test.sh script to do the testing if user pass `setup` option, if user wants to use the existing Codewind user can run test.sh without option `setup`.

Reply:
Also by design, test.sh is the only script that user need to use for testing.

Reply:
understood